README.md

```diff
@@ -1,5 +1,9 @@
 # Bitmovin Player Conviva Analytics Integration
 
+## Compatibility
+**This version of the Conviva Analytics Integration works only with Player Version 7.x.
+The recommended version of the Conviva SDK is 2.146.0.36444.**
```

Comment:
I forgot in the previous review: please add a link to the changelog. Just write `See [CHANGELOG](CHANGELOG.md) for details.`.